### DISCUSSION TITLE
[1 febbraio 2022 - B] Esercizio 5

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#215] [Esami]
**Esercizio 5 (2 punti)**

Si disegni il diagramma temporale dei segnali **y0, y1 e z** del seguente circuito a partire dallo stato **y0=0** e **y1=1** quando riceve in ingresso la sequenza di bit $01011$.

![image](https://user-images.githubusercontent.com/69389461/211826537-2ff650fd-7dba-4e8c-ab2a-7a914877f172.png)

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/216333210-3d19b8ad-1494-46c7-b360-80d638980d96.png)

**Comment by RaulComaz** [comment]
IL penultimo z è sbagliato perchè è =1
anche il disegno della z dovrebbe essere sbagliato per 2 motivi: uno per l'errore di prima e il secondo perchè dovrebbe da 0

**Comment by fraxb04** [comment]
@CiottoloMaggico corretto
@GrenzGaenger450 ti allego il mio esercizio, ed anche se ho scritto la codifica degli stati diversa, torna lo stesso grafico:
![image](https://user-images.githubusercontent.com/122573871/216778795-f4790c8a-c5c3-488a-ad8f-3b3aaab48755.png)
